### DISCUSSION TITLE
Too laggy after StageWebView opens the webpage?

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#2935] [Q&A]
```
import flash.media.StageWebView;

var webView:StageWebView = new StageWebView();
webView.viewPort = new Rectangle(0, 0, 4800,1080);
webView.stage = stage;
webView.loadURL("https://www.720yun.com/vr/a82jOO4uOn7");
```

After opening the special card, browsing through the browser is normal. Did I set it up incorrectly? Or can I choose another browser kernel?
thanks

**Comment by D5CN** [comment]
in PC,default StageWebView is not support HTML5 standard.
you can use WebView2 which runing on Microsoft Edge SDK. Detail is here (https://github.com/airsdk/Adobe-Runtime-Support/discussions/1170#discussioncomment-1346559)
or use ANE just like this:https://github.com/tuarua/WebViewANE

**Reply by D5CN** [reply]
By the way,if you want make a comunication between AS3 and HTML with http request or websocket connection,can use this code ^^
https://github.com/D5CN/ASHttpd

**Reply by Jason0247** [reply]
Thank you very much. I have been feeling a headache about calling HTML.
Especially when calling local HTML

**Reply by shaucky** [reply]
Interesting module, it looks very practical! But it seems to rely on network connectivity… It would be great if there were better ways to interact with JavaScript and ActionScript locally.

**Reply by D5CN** [reply]
If your app work in PC with WebView2 or mobile with ANE just like WebViewANE,you don't need this code .because javascript can comunicate with AS3 by WebView support.But in mobile device with native webview,most of them can not support that.ASHttp can work in these cases.and also you can package your html file with your app,send message to as3 by http request or websocket.

**Comment by Jason0247** [comment]
I would like to express my sincere gratitude to you here!
After using the method provided by (# 1170 (comment)), the speed of opening H5 became normal!
thanks @D5CN
